kanidm_proto/src/v1.rs

```diff
@@ -776,6 +776,7 @@ impl fmt::Display for TotpAlgo {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct TotpSecret {
     pub accountname: String,
+    /// TODO: #860 this'll be domain_display_name
```

Comment:
It still needs to be called issuer though ....

Reply:
Yep, this is just me dropping flags around the code going "hey make changes here"